content/en/news/releases/1.11.x/announcing-1.11/change-notes/index.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Change Notes
+linktitle: 1.11.0
+subtitle: Minor Release
+description: Istio 1.11.0 release notes.
+publishdate: 2020-07-29
+release: 1.11.0
+weight: 10
+aliases:
+    - /news/announcing-1.11.0
+---
+
+{{< warning >}}
+This is an automatically generated rough draft of the release notes and has not yet been reviewed.
+{{< /warning >}}
+
+# Release Notes
+
+## Deprecation Notices
+
+These notices describe functionality that will be removed in a future release according to [Istio's deprecation policy](/about/feature-stages/#feature-phase-definitions). Please consider upgrading your environment to remove the deprecated functionality.
+
+
+
+## Traffic Management
+
+
```

Comment:
Comment here just in case I forget, we will need to add CNI beta promotion here: https://github.com/istio/istio/pull/34178. Still waiting for TOC approval for https://github.com/istio/enhancements/pull/94 before merging the release notes PR. Will update here once merged.

Reply:
- **Promoted** [CNI](/docs/setup/additional-setup/cni/) to beta ([Issue #86](https://github.com/istio/enhancements/issues/86)).